README.md

```diff
@@ -1,8 +1,42 @@
-## Multiverse-Inventories
+<p align="center">
+<img src="https://raw.githubusercontent.com/Multiverse/Multiverse-Core/main/config/multiverse2-long.png" alt="Multiverse Logo">
+</p>
 
-Copyright (c) 2011, The Multiverse Team All rights reserved.
```

Comment:
Make a license file for it

Reply:
Should I use the same LICENCE as MV Core?

Reply:
yea, just make sure core is renamed to inventories